client/style.css

```diff
@@ -51,19 +50,15 @@
   border-style: solid;
   border-width: 2px;
   border-color: #10b535;
-  -webkit-transition-duration: 300ms;
```

Comment:
Just curious, why did you remove the browser polyfills in the last commit?

Reply:
This was a side effect of compiling, I'm not so sure why it did that, but I merely just removed the hover selector.